dts/msm8974/apq8074-samsung-mondrianwifi.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0xB801FF01 0 0>,
+	 <0xB801FF01 7 0>,
+	 <0xB801FF01 8 0>,
+	 <0xB801FF01 12 0>;
+	mondrianwifi {
+		compatible = "samsung,mondrianwifi", "qcom,apq8074", "lk2nd,device";
+		model = "Samsung Galaxy Tab PRO 8.4 WIFI (SM-T320)";
+		lk2nd,match-cmdline = "*samsung.hardware=SM-T320*";
+		lk2nd,smd-rpm-hack-opening;
```

Comment:
Originally this is required on some msm8916/msm8939 samsung devices to boot a kernel. Make sure that you need it, especially when the platform is apq8074 instead.

Reply:
> Make sure that you need it, especially when the platform is apq8074 instead.

Well even with this,  it cant boot to lk2nd.

Reply:
https://github.com/msm8953-mainline/linux/commit/0a4b7f032a6437e812565da35c6e2a4a6cb2a2c2

See this for the details, if you are interested.

Reply:
humm the hack look like it is only gnooa affect booting that kernal so it has nothing to do with my issue (It cant boot into lk2nd).

So maybe my issue is caused by android 4's bl?

Is there any there that i could use to fix this?